main.py

```diff
@@ -1,10 +1,57 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        player_guild = player_data["guild"]
+        player_race = player_data["race"]
+        player_skills = player_data["race"]["skills"]
+
+        if player_guild:
+            if not Guild.objects.filter(name=player_guild["name"]).exists():
+                if player_guild["description"]:
+                    guild_description = player_guild["description"]
+                else:
+                    guild_description = None
+                Guild.objects.create(
+                    name=player_guild["name"],
+                    description=guild_description
+                )
```

Comment:
you can use get_or_create method https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create